### ISSUE TITLE
anonymous formations must lead to warnings

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#35]
This code is not good:

```
malloc.of
  64
  [m]
    QQ.io.stdout > @
      "foo"
```

Instead, should be this:

```
malloc.of
  64
  [m] >>
    QQ.io.stdout > @
      "foo"
```

**Comment by yegor256** [comment]
@maxonfjvipon maybe nothing, in the example above. However, look at this one:

```
[] > app
  "Hello!" > t
  malloc.of > @
    64
    [m]
      QQ.io.stdout > @
        t
```

This will lead to a failure in runtime. Maybe we should warn about anonymous formation, if it uses any objects that are not defined inside it?

**Comment by maxonfjvipon** [comment]
@yegor256 that sounds more reasonable
